.github/workflows/packaging.yaml

```
@@ -114,120 +105,87 @@ jobs:
           tar xf apache-arrow-nanoarrow-${VERSION}.tar.gz
           mv apache-arrow-nanoarrow-${VERSION} nanoarrow
 
-      - name: Show inputs
-        shell: bash
-        run: |
-          echo "upload_artifacts: ${{ inputs.upload_artifacts }}"
-          echo "schedule: ${{ github.event.schedule }}"
-          echo "ref: ${{ github.ref }}"
-
       - name: Build documentation
         run: |
-          pushd nanoarrow
-          docker compose run --rm docs
-          popd
+          echo "::group::Docker Pull"
+          cd nanoarrow
+          docker compose run --rm -e GITHUB_ACTIONS docs
 
```

You are never closing the group, is that intentional?
```suggestion
          echo "::group::Docker Pull"
          cd nanoarrow
          docker compose run --rm -e GITHUB_ACTIONS docs
          echo "::endgroup::"
```